src/app/core/export/query.service.ts

```diff
@@ -307,7 +307,7 @@ export class QueryService {
     key: string,
     value: string
   ): any[] {
-    const values = value.replace(new RegExp(" ", "g"), "").split("|");
+    const values = value.trim().split(/\s*\|\s*/);
```

Comment:
why is the simple `.split("|")` not enough here anymore? (maybe add comment to document this even?)

Reply:
your comment explains the regex (I got that) but not why we need a regex and can't just split at any "|"  ;-)

Reply:
It's just safeguard so both `male|female` as well as `male | female` works. Spaces around the `|` are ignored allowing people to define the query either way.